homeassistant/components/hunterdouglas_powerview/const.py

```diff
@@ -62,7 +62,12 @@
 PV_ROOM_DATA = "pv_room_data"
 COORDINATOR = "coordinator"
 
-HUB_EXCEPTIONS = (ServerDisconnectedError, asyncio.TimeoutError, PvApiConnectionError)
+HUB_EXCEPTIONS = (
+    ServerDisconnectedError,
+    asyncio.TimeoutError,
+    PvApiConnectionError,
+    PvApiResponseStatusError,
```

Comment:
Would this also retry if the status code is something different then 502? If yes, is this intended? If it is, should we change the title of the PR? The error looks like it's thrown for every response status other than 200 OK.

Reply:
https://github.com/sander76/aio-powerview-api/blob/master/aiopvapi/helpers/aiorequest.py#L29

Reply:
We do want to catch them all.  There is no auth here so no risk of catching something we don't want to retry on